app/javascript/angular/code/controllers/_sessions_list_ctrl.js

```diff
@@ -84,9 +86,17 @@ export const SessionsListCtrl = (
   };
 
   $scope.newSessionsForList = function() {
-    return $scope.sessions.get().map(formatSessionForList);
+    return $scope.sessions
+      .get()
+      .map($scope.selectedStream)
+      .map(formatSessionForList);
   };
 
+  $scope.selectedStream = session => ({
```

Comment:
Have you considered not having this on the `$scope`? Could be a vanilla arrow function